### ISSUE TITLE
SearchBar property showLoading not working

### ISSUE THREAD

**Issue by AleksandarSimikic** [react-native-elements/react-native-elements#2524]
Hello.

I am using "react-native-elements" SearchBar component in my App. I have a problem with showLoading property.
I have a state which regulates whether it should be true or false. At first, i thought it was code problem, like state batches updates. But it is not, when i set showLoading property to true, it still wont show spinner.

Everything worked great on this configuration(last version of the app): 
        "react": "16.8",
        "react-native": "0.59.9",
        "react-native-elements": "^1.2.7",
        "react-native-vector-icons": "^6.6.0",

But on new version it doesnt. Latest configuration is:
        "react": "16.13.1",
        "react-native": "0.63.2",        
        "react-native-elements": "^2.3.0",        
        "react-native-vector-icons": "^7.0.0",

I will supply SearchBar code without handlers.
SearchBar: 

               <SearchBar
                    placeholder="Search Country"
                    placeholderTextColor={"#b6b6b6"}
                    inputStyle={{ fontSize: 16, color: "black" }}
                    searchIcon={{ size: 24 }}
                    clearIcon={!this.state.showLoading}
                    showLoading={this.state.showLoading}
                    onChangeText={(text) => this.SearchFilterFunction(text)}
                    onClear={() => this.SearchFilterFunction("")}
                    value={this.state.searchText}
                    inputContainerStyle={{ backgroundColor: "#e8e8e8", borderRadius: 5 }}
                    containerStyle={{
                        backgroundColor: "#fff",
                        borderBottomColor: "#fff",
                        borderTopColor: "#fff",
                        padding: 10,
                    }}
                    autoCorrect={false}
                />

This code worked on previous configuration. (I have also set showLoading to true but it didnt work)

I tried also samples of SearchBar, still there is no spinner.

I would appriciate help.

Thank you!

**Comment by AleksandarSimikic** [comment]
Fixed it.

**Comment by pk936** [comment]
@AleksandarSimikic how did you fix it. ine is not working

**Comment by dave-vazquez** [comment]
I was able to solve this issue with the following implementation:

```javascript
      <SearchBar
        lightTheme
        showLoading
        containerStyle={s.searchContainer}
        inputContainerStyle={s.inputContainer}
        loadingProps={{
          animating: searching,
          color: "black",
        }}
      />
```

I set animating in loadingProps to a boolean state variable, and to get the activity indicator to appear I just pass in a color that will show up with my custom styles.

Note that `animating` is set to true by default. You don't have to set it equal to any particular value, but if you want it to appear conditionally this would be the way to do it.
